osu.Game.Tests/Visual/UserInterface/TestSceneFooterButtonMods.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Osu.Mods;
+using osu.Game.Screens.Select;
+
+namespace osu.Game.Tests.Visual.UserInterface
+{
+    public class TestSceneFooterButtonMods : OsuTestScene
+    {
+        public override IReadOnlyList<Type> RequiredTypes => new[]
+        {
+            typeof(FooterButtonMods)
+        };
+
+        private readonly TestFooterButtonMods footerButtonMods;
+
+        public TestSceneFooterButtonMods()
+        {
+            Add(footerButtonMods = new TestFooterButtonMods());
+        }
+
+        [Test]
+        public void TestIncrementMultiplier()
+        {
+            AddStep(@"Add Hidden", () => changeMods(new Mod[] { new OsuModHidden() }));
+            AddAssert(@"Check Hidden multiplier", () => footerButtonMods.MultiplierText.Text == @"1.06x");
```

Comment:
comparing with strings is clumsy. the mod rates may change and we don't want to have to manually update all these tests.